src/zeep/xsd/types/complex.py

```diff
@@ -247,7 +247,17 @@ def render(self, parent, value, xsd_type=None, render_path=None):
 
         # Render attributes
         for name, attribute in self.attributes:
-            attr_value = value[name] if name in value else NotSet
+            import sys
+            if sys.version_info < (3, 4):
+                if isinstance(value, str) or isinstance(value, unicode):
```

Comment:
an `isinstance(value, basestring)` should be enough here

Reply:
ok, I retry to make new pull request